tests/scripts/run_tests.sh

```diff
@@ -120,6 +120,7 @@ run_frequent_multi_device_pipeline_tests() {
     # Switch to modules only soon
     # run_module_tests "$tt_arch" "llrt" "$pipeline_type"
     ./tests/scripts/run_frequent_regressions_multi_device.sh
+    pytest -svv tests/tt_metal/microbenchmarks -k test_multichip_hop_latency
```

Comment:
Could you please split off the specific microbenchmark into a single pytest file (such as keeping them in one place + calling only one function in a separate file)?

We're trying to redo how tests are run on the infra side, and we think we're going to go with the approach of targeting files and markers. No more `-k`

Reply:
I split them up so now it's a py file for each for the latency and bandwidth benchmarks. I put them in `tests/scripts/run_frequent_regressions_multi_device.sh`  because they (well mainly the latency test) needs t3000.